tests/backends/test_dbmodel.py

```diff
@@ -398,7 +398,7 @@ def test_colocated_db_model_tf(fileutils, wlmutils, mlutils):
     test_script = fileutils.get_test_conf_path("run_tf_dbmodel_smartredis.py")
 
     # Create SmartSim Experience
-    exp = Experiment(exp_name, launcher=test_launcher)
+    exp = Experiment(exp_name, launcher=test_launcher, exp_path=test_dir)
```

Comment:
Al and I found that this is also needed to get the tests to run on horizon. Out of curiosity, do you know what the behaviour is for dashboard if you omit `exp_path`?

Reply:
If you omit `exp_path` and leave have the `SMARTSIM_FLAG_TELEMETRY` flag enabled, it would lead to placement of `.smartsim` directories in the CWD. In order to make sure that these dir always ended up in the `test_output` dir it was decided to give every experiment an appropriate path.